jams/core.py

```diff
@@ -995,6 +995,23 @@ class AnnotationArray(list):
 
     This list subclass provides serialization and search/filtering
     for annotation collections.
+
+    Fancy-indexing can be used to directly search for annotations
+    belonging to a particular namespace.
+
+    Examples
```

Comment:
These examples are helpful - is there a reasonable place where we could document more generically what kind of indexing is allowed?